src/flags/coordination.cpp

```diff
@@ -32,4 +32,6 @@ DEFINE_uint32(instance_health_check_frequency_sec, 1, "The time duration between
 DEFINE_uint32(instance_get_uuid_frequency_sec, 10, "The time duration between two instance uuid checks.");
 // NOLINTNEXTLINE(cppcoreguidelines-avoid-non-const-global-variables)
 DEFINE_string(nuraft_log_file, "", "Path to the file where NuRaft logs are saved.");
+// NOLINTNEXTLINE(cppcoreguidelines-avoid-non-const-global-variables)
+DEFINE_bool(coordinator_use_durability, true, "Whether to use durability for coordinator logs and snapshots.");
```

Comment:
maybe better name: --ha-durability or --use-ha-durability or --durable-ha

Reply:
We have coordinator_use_durability, coordinator_id, coordinator_port, so I guess coordinator_durability should be fine? I went for consistency in that direction, what do you think?

Reply:
ha_durability + env flag missing